token/cli/src/main.rs

```diff
@@ -540,6 +541,67 @@ fn command_accounts(config: &Config, token: Option<Pubkey>) -> CommandResult {
     Ok(None)
 }
 
+fn stringify_ui_token_amount(amount: &UiTokenAmount) -> String {
+    let decimals = amount.decimals as usize;
+    if decimals > 0 {
+        let amount = u64::from_str(&amount.amount).unwrap();
+
+        // Left-pad zeros to decimals + 1, so we at least have an integer zero
+        let mut s = format!("{:01$}", amount, decimals + 1);
+
+        // Add the decimal point (Sorry, "," locales!)
+        s.insert(s.len() - decimals, '.');
+        s
+    } else {
+        amount.amount.clone()
+    }
+}
+
+fn stringify_ui_token_amount_trimmed(amount: &UiTokenAmount) -> String {
+    let s = stringify_ui_token_amount(amount);
+    let zeros_trimmed = s.trim_end_matches('0');
+    let decimal_trimmed = zeros_trimmed.trim_end_matches('.');
+    decimal_trimmed.to_string()
+}
+
+fn command_account(config: &Config, address: Pubkey) -> CommandResult {
+    let account = config
+        .rpc_client
+        .get_token_account_with_commitment(&address, config.commitment_config)?
+        .value
+        .unwrap();
+    println!();
+    println_name_value("Address:", &address.to_string());
```

Comment:
We should take a look and see if `println_name_value` would improve the output of other subcommands (mostly note to self 🙂 )

Reply:
Oh I'm certain that it would!

While we're on the topic, changing the params to be `AsRef<str>` instead of `&str` might improve the ergonomics 😉